libs/vkd3d/resource.c

```diff
@@ -8645,34 +8653,18 @@ static VkMemoryPropertyFlags vkd3d_memory_info_descriptor_heap_memory_properties
     }
 }
 
-static VkMemoryPropertyFlags vkd3d_memory_info_upload_hvv_memory_properties(
+static bool vkd3d_memory_info_decide_hvv_usage(
         const struct vkd3d_memory_topology *topology,
         const struct d3d12_device *device)
 {
-    if (vkd3d_config_flags & VKD3D_CONFIG_FLAG_FORCE_HOST_CACHED)
```

Comment:
For FORCE_HOST_CACHED we don't want to expose ReBAR at all. Mostly to avoid RenderDoc captures being unusably slow.

Reply:
Or maybe this is resolved below ...